cmd/zetaclientd/start.go

```diff
@@ -157,6 +160,25 @@ func start(_ *cobra.Command, _ []string) error {
 
 	go zetacoreClient.UpdateAppContextWorker(ctx, appContext)
 
+	keygen := appContext.GetKeygen()
+	cfg.WhitelistedPeers = []string{}
+	for _, pk := range keygen.GranteePubkeys {
+		pk, err := legacybech32.UnmarshalPubKey(legacybech32.AccPK, pk)
+		if err != nil {
+			return err
+		}
+		bz := pk.Bytes()
+		k, err := crypto2.UnmarshalSecp256k1PublicKey(bz)
+		if err != nil {
+			return err
+		}
+		pid, err := peer.IDFromPublicKey(k)
+		if err != nil {
+			return err
+		}
+		cfg.WhitelistedPeers = append(cfg.WhitelistedPeers, pid.String())
```

Comment:
maybe store this on the go-tss side (under conversion) so we can use it in go-tss tests.